filament/backend/src/vulkan/platform/VulkanPlatform.cpp

```diff
@@ -552,12 +556,18 @@ Driver* VulkanPlatform::createDriver(void* sharedContext,
         mImpl->mDevice = scontext->logicalDevice;
         mImpl->mGraphicsQueueFamilyIndex = scontext->graphicsQueueFamilyIndex;
         mImpl->mGraphicsQueueIndex = scontext->graphicsQueueIndex;
+
+        mImpl->mSharedContext = true;
     }
 
     VulkanContext context;
 
-    auto instExts = getInstanceExtensions();
-    instExts.merge(getRequiredInstanceExtensions());
+    ExtensionSet instExts;
+    // If using a shared context, we do not assume any extensions.
```

Comment:
should we getting the extensions that are available from the shared context instead? does it matter?